src/seedu/addressbook/parser/Parser.java

```diff
@@ -246,6 +246,10 @@ private Command prepareFind(String args) {
 
         // keywords delimited by whitespace
         final String[] keywords = matcher.group("keywords").split("\\s+");
+        for (int i=0; i<keywords.length; i++) {
```

Comment:
Coding standard violations in the for statement.

for (int i = 0; i < keywords.length; i++) {